acbuild/acbuild.go

```diff
@@ -249,7 +249,10 @@ func main() {
 	// Make help just show the usage
 	cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`)
 
-	cmdAcbuild.Execute()
+	err := cmdAcbuild.Execute()
```

Comment:
As far as I can tell errors here are directly from cobra (for example, if the called subcommand doesn't exist) and cobra will have already printed out an error and/or help page by this point.